src/main/java/zhenya/workshop_5/Browser.java

```diff
@@ -0,0 +1,26 @@
+package zhenya.workshop_5;
+
+public class Browser extends BasePage {
```

Comment:
Browser is not related to pages, it's totally different entity, so we do not need here a constructor and inheritance